# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/websocket/WebSocketChannel.cpp
# Commit: 42af2da59248
# Full Hash: 42af2da59248c7d0dbfab1ee3e21a5b5350b31d2
# Author: Kershaw Chang <kershaw@mozilla.com>
# Date: 2022-03-23 21:48:10
# Description:
#   Bug 1760894 - Change MOZ_RELEASE_ASSERT back to MOZ_ASSERT, r=necko-reviewers,dragana
#   
#   This is a wallpaper patch, not a real fix.
#   This assertion was accidentally changed to `MOZ_RELEASE_ASSERT` in D135727.
#   
# ==============================================================================

diff -r e2acbe11b6a2 -r 42af2da59248 netwerk/protocol/websocket/WebSocketChannel.cpp
--- a/netwerk/protocol/websocket/WebSocketChannel.cpp	Wed Mar 23 12:03:28 2022 +0000
+++ b/netwerk/protocol/websocket/WebSocketChannel.cpp	Wed Mar 23 12:20:30 2022 +0000
@@ -3265,8 +3265,8 @@
     AbortSession(NS_ERROR_NET_TIMEOUT_EXTERNAL);
     // mReconnectDelayTimer is only modified on MainThread
   } else if (timer == mReconnectDelayTimer) {
-    MOZ_RELEASE_ASSERT(mConnecting == CONNECTING_DELAYED,
-                       "woke up from delay w/o being delayed?");
+    MOZ_ASSERT(mConnecting == CONNECTING_DELAYED,
+               "woke up from delay w/o being delayed?");
     MOZ_ASSERT(NS_IsMainThread(), "not main thread");
 
     {
